modules/gapi/test/common/gapi_tests_common.hpp

```diff
@@ -272,6 +349,13 @@ struct TestWithParamBase : TestFunctional,
         Fixture() { InitF(type, sz, dtype); } \
     };
 
+#define GAPI_TEST_FIXTURE_NO_INIT(Fixture, API, Number, ...) \
```

Comment:
Please replace GAPI_TEST_FIXTURE_NO_INIT macro and all code base developed to support it with the appropriate feature from another your [#16788](https://github.com/opencv/opencv/pull/16788/files) PR once you are done with one.

Reply:
done